packages/portfolio-frontend/project.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@promethean/portfolio-frontend",
+  "root": "packages/portfolio-frontend",
+  "targets": {
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "pnpm run build",
+        "cwd": "packages/portfolio-frontend"
+      },
+      "outputs": ["{projectRoot}/dist"]
+    },
+    "lint": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "pnpm exec eslint .",
+        "cwd": "packages/portfolio-frontend",
+        "forwardAllArgs": false
+      }
```

Comment:
Same here: swap to `pnpm run lint` for symmetry across packages and to keep eslint flags contained in the script.